pkgs/servers/overseerr/default.nix

```diff
@@ -0,0 +1,91 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, mkYarnPackage
+, nodejs
+, sqlite
+, fetchYarnDeps
+, python3
+, pkg-config
+, glib
+}:
+
+let
+  pin = lib.importJSON ./pin.json;
```

Comment:
Wouldn't this result in import from derivation? Maybe it's better to have the update script generate a nix file containing an attrset, and do this with `pin = import ./pin.nix`?